src/mock/v1.2/intent-b.ts

```diff
@@ -13,8 +13,8 @@ onFdc3Ready().then(async () => {
         return context;
     });
 
-    //broadcast that intent-a has opened
+    //broadcast that intent-b has opened
     await sendContextToTests({
-        type: "fdc3-intent-a-opened"
+        type: "fdc3-intent-b-opened"
```

Comment:
is this a bug in the test that @ksgeorgieva is fixing, @Joe-Dunleavy ?

Reply:
I think this was accidentally introduced in the latest version while refactoring and copy/pasting that function from intent-a.ts